lib/fact_changes.rb

```diff
@@ -1,32 +1,28 @@
 require 'token_util'
 require 'changes_support/disjoint_list'
 require 'changes_support/transaction_scope'
-require 'google_hash'
 
 class FactChanges
```

Comment:
I'm a bit more comfortable with this than the disjoint list, but still a little uncertain when it comes to adding documentation.

Some of the terminology in the class seemed a little muddled. For example, I'm not quite sure why predicate is so named, as it seems to be associated with a value that isn't a bool.

Reply:
```predicate``` is in the meaning of a ```verb``` in a sentence. Using the analogy to a sentence with subject, predicate and object:
subject -> It will be the asset (DN1234)
predicate -> property ('a' ~= equivalent to saying 'is a')
object -> value ('Tube')

Reply:
Ahh, the poor state of British grammar education rear its head.